### PR TITLE
cg_tutorial: display the medkit and ammo feedback in orange

### PR DESCRIPTION
**Pull request by illwieckz**
Not perfect, but better than what we have.

[tutorial medkit red message](https://dl.illwieckz.net/b/unvanquished/bugs/tutorial-medkit-red-message/unvanquished_2021-11-02_235331_000.jpg)

[tutorial medkit red message](https://dl.illwieckz.net/b/unvanquished/bugs/tutorial-medkit-red-message/unvanquished_2021-11-02_235906_000.jpg)

### CODE REVIEW

File: src/cgame/cg_tutorial.cpp [hunk 1]
```diff
@@ -159,6 +159,8 @@ static entityState_t *CG_BuildableInRange( playerState_t *ps, float *healthFract
 	return &es;
 }
 
+#define COLOR_ALARM "^1" // Red
```
Comment (necessarily-equal) [inline src/cgame/cg_tutorial.cpp:162]: In orange maybe? 

```suggestion
#define COLOR_ALARM "^8" // Red
```

Otherwise, LGTM

**Review by necessarily-equal — APPROVED** [review]

File: src/cgame/cg_tutorial.cpp (hunk 1 above)
Comment (bmorel) [reply src/cgame/cg_tutorial.cpp:162]: Instead of discussing colors, make it a client side cvar and call it a day? In any any cases, LGTM too.

**Comment by bmorel** [comment]
Some thinking (I'm still fully ok to merge this in current state): one step further would likely be to apply color to the medkit icon too, so that even without tutorial shown, the info would be there (I know that I regularly try to use absent medkit, just in case I still have it, and basically use the warning message to know if it worked or not...).
I now know how to do that (my solution is probably not perfect though, since it implies to have one file per color variant, which is suboptimal), and will send a PR when this one will be merged, so that I can use the same color.

Also, the ammo color change should likely be done the same moment the «beacon hint» is shown, maybe with several colors, like, yellow when low on ammo, and red when there's no ammo remaining at all (current behavior).
This would not help when tutorial is not shown, though, for that, ideally the ammo and clip bars should change color or, maybe even start blinking?

**Comment by bmorel** [comment]
Any reason to not merge this?

**Comment by slipher** [comment]
LGTM

File: src/cgame/cg_tutorial.cpp (hunk 1 above)
Comment (illwieckz) [reply src/cgame/cg_tutorial.cpp]: > In orange maybe?

Good idea, it may be more readable, done.

**Comment by illwieckz** [comment]
> Also, the ammo color change should likely be done the same moment the «beacon hint» is shown, maybe with several colors, like, yellow when low on ammo, and red when there's no ammo remaining at all (current behavior).

I have looked at it at the time I submitted the PR and it looks like detecting a weapon is low on ammo is tricky (especially, it's per-weapon specific), while “empty” is universal. Of course more can be done later.

**Pull request merged**